docs/source/model/signatures.rst

```diff
@@ -158,11 +158,20 @@ Column-based signature also support composite data types of these primitives.
 - Array (list, numpy arrays)
 - Spark ML vector (it inherits ``Array[double]`` type)
 - Object (dictionary)
+- AnyType
```

Comment:
Can we add mentions (with links to this section) within the guides and tutorials associated with signature definitions for custom GenAI models? 
This is a super-important feature that should be discoverable from many different places :)